errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/DataBindingUtil.java

```diff
@@ -467,4 +467,12 @@ public static boolean isBindableType(final MetaClass type) {
 
     return bindableTypes;
   }
+
+  private static boolean validateWildcard(MetaClass bindable) {
+    if(bindable.isFinal()) {
+      log.info("@Bindable types cannot be final, ignoring: " + bindable.getFullyQualifiedName());
```

Comment:
Same comment as the above logging: we probably don't want to spam the logs with non-bindable types. Could you please change this to debug level?

Also please see the above logging comment for how you can avoid the String concatenation when this message isn't printed.